manage_breast_screening/notifications/management/commands/create_reports.py

```diff
@@ -5,16 +5,16 @@
 from django.core.management.base import BaseCommand
 from django.db import connection
 
-from manage_breast_screening.notifications.management.commands.helpers.exception_handler import (
-    exception_handler,
+from manage_breast_screening.notifications.management.commands.helpers.command_handler import (
+    CommandHandler,
 )
 from manage_breast_screening.notifications.models import ZONE_INFO
 from manage_breast_screening.notifications.queries.helper import Helper
 from manage_breast_screening.notifications.services.blob_storage import BlobStorage
 from manage_breast_screening.notifications.services.nhs_mail import NhsMail
 
 logger = getLogger(__name__)
-INSIGHTS_ERROR_NAME = "CreateReportsError"
+INSIGHTS_JOB_NAME = "CreateReports"
```

Comment:
This is when I miss Ruby. Feels like metaprogramming in the contextmanager would solve this problem nicely for all job names.